### PR TITLE
Chapter1 exercise done, first implementation

### PR DESCRIPTION
**Pull request by pkwiecienJ8**
### Solutions for Chapter 1

cc @vrom911 @chshersh

### CODE REVIEW

**Comment by pkwiecienJ8** [comment]
Dont know why by it seems some tests are failing. And on my local machine everythin is passing expect of last test.

File: src/Chapter1.hs [hunk 1]
```diff
@@ -490,7 +490,8 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = (abs n) `mod` 10
```
Comment (chshersh) [inline src/Chapter1.hs:494]: You can skip `()` here 🙂 
```suggestion
lastDigit n = abs n `mod` 10
```

File: src/Chapter1.hs [hunk 2]
```diff
@@ -568,7 +573,10 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c 
+    | c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u' = True
```
Comment (chshersh) [inline src/Chapter1.hs:578]: Absolutely correct!  👍🏻 

As `||` alread returns a value of type `Bool`, you don't need to use guards and return the result of operator application directly:

```haskell
isVowel c =
        c == 'a' || c == 'A'
    ||  c == 'e' || c == 'E'
    || ...
```

**Review by chshersh — APPROVED** [review]
The implementation looks great! 👏🏻 
Feel free to merge this PR 🏁

**Comment by chshersh** [comment]
@pkwiecienJ8 Can you point to the failing test? I see that tests for Chapter One are passing on CI.

**Comment by pkwiecienJ8** [comment]
> @pkwiecienJ8 Can you point to the failing test? I see that tests for Chapter One are passing on CI.

I think I made mistake, you are right. Sorry for bothering :/

**Pull request merged**